src/Pelagos/Entity/ResearchGroup.php

```diff
@@ -86,6 +86,21 @@ class ResearchGroup extends Entity
      */
     protected $name;
 
+    /**
+     * Name of a research group.
+     *
+     * @var string $name
+     *
+     * @access protected
+     *
+     * @ORM\Column(type="citext", unique=true, nullable=true)
```

Comment:
Citext is case insensitive text, also uniqueness must be enforced.

Reply:
I tested and it does recognize "oyster" and "Oyster" as being the same, which I think is the right behavior for uniqueness. (the violation just needs to be handled)

Reply:
Added the unique constraint validator